README.md

```diff
@@ -0,0 +1,151 @@
+# aws-cli
+
+[![Build Status](https://travis-ci.org/aws/aws-cli.svg?branch=develop)](https://travis-ci.org/aws/aws-cli)
+
+[![Gitter](https://badges.gitter.im/aws/aws-cli.svg)](https://gitter.im/aws/aws-cli)
+
+This package provides a unified command line interface to Amazon Web Services.
```

Comment:
Just as a reminder, we should add the note about finding information about v2 at the top here.